core/templates/pages/topic-viewer-page/practice-tab/practice-tab.directive.ts

```diff
@@ -43,12 +43,43 @@ angular.module('oppia').directive('practiceTab', [
           var ctrl = this;
 
           ctrl.newPracticeSession = function() {
+            var commaSeparatedSelectedSubtopics = '';
+            for (var idx in ctrl.selectedSubtopicIndices) {
+              if (ctrl.selectedSubtopicIndices[idx]) {
+                commaSeparatedSelectedSubtopics += (
+                  ctrl.availableSubtopics[idx].getTitle() + ',');
```

Comment:
May need to be careful about encoding here. (Perhaps create the list and encode as json string?)

Reply:
Done, made it a query string with stringified array, read using json.loads in the backend.